### PR TITLE
use "Next-Gen"

### PR DESCRIPTION
**Pull request by nadinequinr3**
Preview here https://docs.preview.docs.r3.com/

### CODE REVIEW

**Review by yarivus — COMMENTED** [review]
LGTM 
Although I'm confused why Beta2 == NextGen as opposed to Corda5 == NextGen 
And if so, it should be Corda Next-Gen: Beta 2 (as it's the 2nd beta of Next-Gen Corda)

**Comment by nadinequinr3** [comment]
> LGTM Although I'm confused why Beta2 == NextGen as opposed to Corda5 == NextGen And if so, it should be Corda Next-Gen: Beta 2 (as it's the 2nd beta of Next-Gen Corda)

Good point. I'll tweak. Thanks.

**Review by paulmoloneyr3 — DISMISSED** [review]

**Review by yarivus — APPROVED** [review]
LGTM

File: content/en/platform/corda/5.0-beta/_index.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ menu:
     name: Corda 5.0 Beta 2
 ---
 
-## Corda 5.0 Beta 2
+## Corda 5.0: Next-Gen Corda - Beta 2
```
Comment (milan-khan) [inline content/en/platform/corda/5.0-beta/_index.md:16]: This seems a little clumsy to me, almost as though we're naming something twice. I think it should be "Next-Gen Corda - Beta 2"

File: themes/doks/layouts/index.html [hunk 2]
```diff
@@ -40,12 +40,12 @@ <h3 class="card-title">Get Started</h3>
         <div class="col">
           <div class="card h-100">
             <div class="card-body">
-              <h3 class="card-title">Corda 5.0 Beta</h3>
-              <p>Build and deploy CorDapps locally with this preview version of Corda 5. Experiment with the new modular
+              <h3 class="card-title">Corda 5.0: Next-Gen Corda - Beta 2</h3>
```
Comment (milan-khan) [inline themes/doks/layouts/index.html:43]: We should say "Corda 5.0 - Beta 2" or "Next-Gen Corda - Beta 2"

**Review by milan-khan — CHANGES_REQUESTED** [review]
We should say "Corda 5.0 - Beta 2" or "Next-Gen Corda - Beta 2"

**Comment by nadinequinr3** [comment]
> We should say "Corda 5.0 - Beta 2" or "Next-Gen Corda - Beta 2"

I was trying to make the link that Corda 5 and Corda Next-Gen are the same things.

**Comment by colin-hillis** [comment]
In principle, using Next-Gen as the name (albeit temporary) is confusing and it will be for customers too. It would have been more sensible to have 'Next-Gen' worked into a tag line for Beta 2.0, but not the name itself. I get it that the new product requires marketing for certain types of communication, but for the technical docs, it is 100% going to be confusing for customers to change from **Corda Next Gen** to **Corda 5.0** in the GA release, or even GA+1. 

If it is the case that we have to run with this, I am not sure if this is the extent of the locations where it is to be changed....perhaps we can keep it to as few as possible, so that there are not dozens of locations where it has to be manually changed when we revert back to Corda 5 again. 

Whatever is decided on the wording front, I think it is important to also include 5 or 5.0 to reduce the chance of confusion, when we decide to change the name back to 5 or 5.0 again.

**Comment by milan-khan** [comment]
@colin-hillis @nadinequinr3 yes "Next-Gen Corda 5.0 - Beta 2.0" works. I think we can have it in the one place i.e. in this landing page. It looks a bit long-winded to be using in many places.

**Review by yarivus — APPROVED** [review]
LGTM

**Review by paulmoloneyr3 — APPROVED** [review]

**Review by colin-hillis — APPROVED** [review]
LGTM

**Pull request merged**